sphinx/api/configuration/suite.rst

```diff
@@ -20,12 +20,12 @@ A suite directory may contain the following:
   modified INI format described above. It describes the suite's purpose and
   identity, e.g. the title, the description, the owner, the access control
   list, and other information. Apart from a few standard fields, a suite is
-  free to store any information in this file. See :rose:file:`rose-suite.info`.
+  free to store any information in this file. See :rose:file:`rose-suite.info`
   for detail.
 * An ``app/`` directory of application configurations used by the suite.
 * A ``bin/`` directory of scripts and utilities used by the suite.
 * An ``etc/`` directory of other configurations and resources used the suite.
-  E.g. fcm make configurations.
+  E.g. ``fcm_make`` configurations.
```

Comment:
This can be `` :rose:app`fcm_make` `` :rose:

Reply:
Well-spotted! Will change. :sunflower: :hibiscus: :tulip: :blossom: